docs/modules/ROOT/pages/deployment-distributed.adoc

```diff
@@ -1,8 +1,6 @@
-= Distributed deployment
+= Distributed multi-cluster deployment
```

Comment:
"multi-cluster" suggests that 0.1 only supports one CBS cluster, when we support multiple (all monitored from the same CMOS instance) - we should make it clearer what "cluster" refers to here

Reply:
@markspolakovs make sense. I'll update it. Thanks!